src/default.js

```diff
@@ -93,8 +95,12 @@ Vue.component('drop', Drop);
                 router,
                 render: h => h(Phoenix)
             });
-            // inject custom config into vuex base state
+            // inject custom config into vuex
             store.dispatch('loadConfig', config.data)
+            // inject custom theme config into vuex
+            let theme = Axios.get(`src/themes/${config.data.theme}.json`).then(res => {
```

Comment:
src is not deployed - right - needs to be at a different location afaik

Reply:
is now themes/owncloud.json for example